policy/Azure-Proactive-Resiliency-Library-v2/azapi/web/azapi_web_serverfarms_use_standard_or_premium_tier.rego

```diff
@@ -0,0 +1,19 @@
+package Azure_Proactive_Resiliency_Library_v2
+
+import rego.v1
+
+
+
+valid_azapi_web_serverfarms_use_standard_or_premium_tier(resource) if {
+    valid_sku_types := {"premium", "standard", "isolatedv2"}
+    some word in valid_sku_types
+    contains(resource.values.body.sku.tier, word)
```

Comment:
Should align with [the ARG Query](https://azure.github.io/Azure-Proactive-Resiliency-Library-v2/azure-resources/Web/serverFarms/#use-standard-or-premium-tier) here:

```rego
contains(lower(resource.values.body.sku.tier), word)
```